project/src/World/DockingPoint/DockingPoint.gd

```diff
@@ -44,11 +46,14 @@ func _ready() -> void:
 func set_docking_remote(node: Node2D, docker_distance: float) -> void:
 	remote_rig.global_rotation = GSAIUtils.vector2_to_angle(node.global_position - global_position)
 	remote_transform.position = docking_point_edge + Vector2.UP * (docker_distance / scale.x)
-	remote_transform.remote_path = node.get_path()
 
+	player_rotation_transform.position = docking_point_edge + Vector2.UP * (docker_distance / scale.x)
+	player_rotation_transform_rig.global_rotation = remote_rig.global_rotation
```

Comment:
As we're removing the old rig, you should set its global_rotation as the old rig did with `GSAIUtils.vector2_to_angle(node.global_position - global_position)`

Reply:
bfeb0e7a7b8052e2fe297833efa58838bcc84abd